scripts/vscode-build.ps1

```diff
@@ -12,7 +13,8 @@ $fileInfo = Get-ItemProperty $filePath
 $projectFileInfo = Get-ProjectFile $fileInfo
 if ($projectFileInfo) {
   $buildTool = InitializeBuildTool
-  $buildArgs = "$($buildTool.Command) -v:m -m -p:UseRoslynAnalyzers=false -p:GenerateFullPaths=true $($projectFileInfo.FullName)"
+  $frameworkArg = if ($framework) { " -p:TargetFramework=$framework" } else { "" }
```

Comment:
```suggestion
  $frameworkArg = if ($framework -ne "") { " -p:TargetFramework=$framework" } else { "" }
```

The code will work as written because `""` is falsy in powershell for strings but helps to be explicit here.